CrocoDash/case.py

```diff
@@ -579,47 +594,65 @@ def process_forcings(
         if process_param_changes:
             self._update_forcing_variables()
 
+    def process_bgc_ic(self):
+        dest_path = self.inputdir / "ocnice" / Path(self.marbl_ic_filepath).name
+        shutil.copy(self.marbl_ic_filepath, dest_path)
+        self.marbl_ic_filename = Path(self.marbl_ic_filepath).name
+
     def process_bgc_iron_forcing(self):
         # Create coordinate variables
-        nx = self.ocn_grid.nx 
+        nx = self.ocn_grid.nx
         ny = self.ocn_grid.ny
         depth = 103
         depth_edges = depth + 1
+        dz = 6000.0 / depth
+        DEPTH = np.linspace(dz / 2, 6000.0 - dz / 2, depth)
+        DEPTH_EDGES = np.linspace(0, 6000, depth_edges)
         ds = xr.Dataset(
             {
-                "DEPTH": (["DEPTH"], np.linspace(0, 6000, depth)),
-                "DEPTH_EDGES": (["DEPTH_EDGES"], np.linspace(0, 6000, depth_edges)),
-                "FESEDFLUXIN": (["DEPTH", "ny", "nx"], np.zeros((depth, ny, nx), dtype=np.float32)),
-                "FESEDFLUXIN_oxic": (["DEPTH", "ny", "nx"], np.zeros((depth, ny, nx), dtype=np.float32)),
-                "FESEDFLUXIN_reduce": (["DEPTH", "ny", "nx"], np.zeros((depth, ny, nx), dtype=np.float32)),
+                "DEPTH": (["DEPTH"], DEPTH),
+                "DEPTH_EDGES": (["DEPTH_EDGES"], DEPTH_EDGES),
+                "FESEDFLUXIN": (
+                    ["DEPTH", "ny", "nx"],
+                    np.zeros((depth, ny, nx), dtype=np.float32),
+                ),
+                "FESEDFLUXIN_oxic": (
+                    ["DEPTH", "ny", "nx"],
+                    np.zeros((depth, ny, nx), dtype=np.float32),
+                ),
+                "FESEDFLUXIN_reduce": (
+                    ["DEPTH", "ny", "nx"],
+                    np.zeros((depth, ny, nx), dtype=np.float32),
+                ),
```

Comment:
I think I pointed out the attributes in the previous review, but these variables can be removed from the dataset (and it looks like the attributes are still being defined below)

Reply:
Sorry, I missed the comment you added. I do think it's okay to remove these; when we add in the actual forcing we can talk about whether we want to bring them back or not. The proposed forcing in https://github.com/marbl-ecosys/MARBL/pull/464 separates the oxidized and reduced fluxes into two files rather than adding them together for a total flux, and the vent flux isn't split in the same way so it's a little misleading to see those terms in the file.